lib/item_repository.rb

```diff
@@ -0,0 +1,55 @@
+require "item"
+
+class ItemRepository
+
+  def all
+    sql = 'SELECT id, name, unit_price, quantity FROM items;'
+    result_set = DatabaseConnection.exec_params(sql,[])
+
+    items = []
+    result_set.each do |smth|
+      item = Item.new
+      item.id = smth['id'].to_i
+      item.name = smth['name']
+      item.unit_price = smth['unit_price']
```

Comment:
I learned that "name" is reserved in Postgresql so I recommend you use another definition for that, like title.